### PR TITLE
Update danish translation - 1.0.5

### PR DESCRIPTION
**Pull request by Stisen1**
Focusing on translating:
- town building descriptions and windows.
- diplomacy windows
- single player scenario window

Anything that has to do with campaign translations is secondary.

### CODE REVIEW

File: files/lang/dk.po [hunk 1]
```diff
@@ -3951,6 +3912,9 @@ msgid ""
 "switching into the 'Hot Seat' when it is their turn."
 msgstr ""
 
+msgid "Cancel back to the main menu."
+msgstr "Afbryd og gå tilbage til hovedmenuen."
+
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:3917]: It's in this file version... in a new spot though. I have NO IDEA why strings removes them selves! I use PO-edit for translations, but maybe my conflict resolving is flawed... Or is it when I change between "UTP-8" and "CP1252"?

File: files/lang/dk.po [hunk 2]
```diff
@@ -4578,6 +4542,9 @@ msgstr ""
 msgid "Keyboard|SPACE"
 msgstr ""
 
+msgid "Keyboard|LANG"
+msgstr ""
+
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:4547]: Donno... How can I know whether it exits or not in the so-called master? Is the master something else than my v.1.0.4 
 po.dk?

File: files/lang/dk.po [hunk 3]
```diff
@@ -5987,11 +5923,16 @@ msgstr ""
...
 msgid "Enemy heroes are now fully identifiable."
 msgstr ""
 
-msgid "Identify Hero"
-msgstr "Identificer helt"
+msgid ""
+"You must be within %{count} spaces of a monster for the Visions spell to "
+"work."
+msgstr ""
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:5935]: I DON'T MOVE ANYTHING! What would the purpose be? I wish the program would prompt me about something as silly as that... Po-Edit move things around on the UI-surface, but does this also affect string-id's positions?

File: files/lang/dk.po [hunk 4]
```diff
@@ -2258,56 +2265,81 @@ msgid ""
...
+"Voldgraven forsinker angribende tropper. Enhver trop, som træder ned i "
+"voldgraven, ender sin tur dér. Så længe den befinder sig her, vil den "
+"desuden være mere sårbar overfor angreb."
 
 msgid ""
 "The Castle improves town defense and increases income to %{count} gold per "
 "day."
 msgstr ""
+"Borgen forstærker byens forsvar mod angreb. Borgen øger desuden dit riges "
+"indkomst med %{count} om dagen."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:2320]: yes "guldstykker"

File: files/lang/dk.po [hunk 5]
```diff
@@ -5893,11 +5937,6 @@ msgid ""
 "work."
 msgstr ""
 
-msgid ""
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:5896]: Moved mising lines from another location to this position

File: files/lang/dk.po [hunk 6]
```diff
@@ -5987,12 +6026,20 @@ msgstr ""
 msgid "%{spell} failed!!!"
 msgstr ""
 
+msgid "This spell is already in use."
```
[the remark this answers is not in the document]
Comment (Stisen1) [reply files/lang/dk.po:6029]: If it is "this spell is already in use." you mean, I have now removed it. 

It already exists in another string-location

File: files/lang/dk.po [hunk 7]
```diff
@@ -2258,56 +2265,81 @@ msgid ""
 "can also provide scouting information on enemy towns. Additional Guilds "
 "provide more information."
 msgstr ""
+"Tyvelauget skaffer informationer om hvordan andre spillere klarer sig i "
+"forhold til dig. Desuden udspejdes fjendtlige byer og borge også. Jo flere "
+"Tyvelaug du har, jo flere information får du (maks 5)."
 
 msgid "The Tavern increases morale for troops defending the castle."
-msgstr ""
+msgstr "Kroen øger de forsvarende styrkers moral med 2 point."
```
Comment (ihhub) [inline files/lang/dk.po:2273]: Interesting point: ideally in English we should state the same. What do you think @zenseii ?

File: files/lang/dk.po [hunk 8]
```diff
@@ -2258,56 +2265,81 @@ msgid ""
 "can also provide scouting information on enemy towns. Additional Guilds "
 "provide more information."
 msgstr ""
+"Tyvelauget skaffer informationer om hvordan andre spillere klarer sig i "
+"forhold til dig. Desuden udspejdes fjendtlige byer og borge også. Jo flere "
+"Tyvelaug du har, jo flere information får du (maks 5)."
```
Comment (ihhub) [inline files/lang/dk.po:2270]: I am not sure that stating the maximum amount of Thieves' Guilds is important (the same actually applies for Marketplaces). As you might noticed the game barely have any numbers in messages which on one side is cool forcing people to read text without focusing on number. On the other hand it is hard to know without prior experience.

If it is not critical I suggest to follow the original game texts without specifying numbers.

**Review by ihhub — COMMENTED** [review]
Hi @Stisen1 , I left just 2 comments here. Could you please check them?

File: files/lang/dk.po (hunk 7 above)
Comment (Stisen1) [reply files/lang/dk.po:2273]: I have left this line as it is, if your excellencies allow it.

File: files/lang/dk.po (hunk 8 above)
Comment (Stisen1) [reply files/lang/dk.po]: I have now removed "(maks 5)" from thieves guild and marketplace descriptions

**Review by ihhub — APPROVED** [review]

**Comment by ihhub** [comment]
@Stisen1 , thank you so much for this update!

**Pull request merged**